frontend/e2e/page-objects/InquiryPage.ts

```diff
@@ -0,0 +1,75 @@
+import { expect } from "@playwright/test"
+import HelperBase from "./helperBase"
+
+export default class InquiryPage extends HelperBase {
+  async navigateToInquiriesPage() {
+    await this.page.waitForSelector("p.chakra-text.css-0", { timeout: 5000 })
+    await expect(this.page.locator("p.chakra-text.css-0")).toHaveText(
+      "Welcome back, nice to see you again!",
+    )
+    await this.page.getByText("Inquiries").click()
+  }
+
+  async openInquiryForm() {
+    const inquiryTitle = this.page.getByRole("heading", {
+      name: "Inquiries Management",
+    })
+    await expect(inquiryTitle).toHaveText("Inquiries Management")
+    await this.page.getByRole("button", { name: "Inquiry" }).click()
+    await this.page.waitForSelector(".chakra-modal__footer", {
+      state: "visible",
+    })
+    await expect(
+      this.page.getByLabel("Add Inquiry").getByText("Add Inquiry"),
+    ).toHaveText("Add Inquiry")
+  }
+
+  async addInquiry_whenValidInquiryIsAdded_shouldShowInquiryInInquiriesList(
+    inputText: string,
```

Comment:
question: so how does this test get executed by playwright? Would it automatically create different input text?